### DISCUSSION TITLE
❗ HELP WANTED: Solve the Unsolved! 🔍

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#49017] [Pull Requests]
## Got your Thinking Cap On 🧢❓​​
Hey there! 👋 Interested in contributing to our community? Here are a few questions in our category from the last two weeks that could still use an answer. Take a look 👀 :

- [Relative date on commits/prs show incorrectly starting February 1 2023](https://github.com/orgs/community/discussions/48853)
- [Checks failed before Status Check timeout reached](https://github.com/orgs/community/discussions/48723)
- [How to filter out review requests from codeowner groups?](https://github.com/orgs/community/discussions/48031)
- [Checks failed before Status Check timeout reached](https://github.com/orgs/community/discussions/48723)
- [I cannot delete a person in Contributors](https://github.com/orgs/community/discussions/48717)

Got an unanswered question you’d like to spotlight? Comment it below!

**Comment by JXXL888** [comment]
hello，A colleague's account could not be logged in and requested to be processed, the official reply email gave the ticket number 2066354 to come here to request processing

**Reply by queenofcorgis** [reply]
Hi there - can you elaborate on the request processing please? The Community is separate from support and generally cannot assist with account-specific issues.
